SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -67,12 +67,12 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 		const FClassInfo* Info = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
 		check(Info);
 
-		const TArray<FString> WorkerAttributes = NetDriver->Connection->GetWorkerAttributes();
+		const TArray<FString>& WorkerAttributes = NetDriver->Connection->GetWorkerAttributes();
```

Comment:
Drive-by changes to the code from https://github.com/spatialos/UnrealGDK/pull/570. I could move this to a separate PR if anyone feels strongly about it.